README.md

```diff
@@ -10,4 +10,4 @@ Yet Another React UI Library
     Primary Alert
 </Alert>
 ```
-
+A Project based on React for Delhi-NCR
```

Comment:
Please remove the demo code and add complete details for Readme, not just one line!